web/templates/admin/job/form.html.eex

```diff
@@ -1,68 +1,52 @@
-<%= form_for @changeset, @action, fn f -> %>
+<%= form_for @changeset, @action, fn form -> %>
   <%= if @changeset.action do %>
     <div class="alert alert-danger">
       <p>Oops, something went wrong! Please check the errors below.</p>
     </div>
   <% end %>
 
   <div class="form-group">
-    <%= label f, :title, class: "control-label" %>
-    <%= text_input f, :title, class: "form-control" %>
-    <%= error_tag f, :title %>
+    <%= input form, :title %>
   </div>
 
   <div class="form-group">
-    <%= label f, :company, class: "control-label" %>
-    <%= text_input f, :company, class: "form-control" %>
-    <%= error_tag f, :company %>
+    <%= input form, :company %>
   </div>
 
   <div class="form-group">
-    <%= label f, :contact, class: "control-label" %>
-    <%= text_input f, :contact, class: "form-control" %>
-    <%= error_tag f, :contact %>
+    <%= input form, :contact %>
   </div>
 
   <div class="form-group">
-    <%= label f, :city, class: "control-label" %>
-    <%= text_input f, :city, class: "form-control" %>
-    <%= error_tag f, :city %>
+    <%= input form, :city %>
   </div>
 
   <div class="form-group">
-    <%= label f, :company_url, class: "control-label" %>
-    <%= text_input f, :company_url, class: "form-control" %>
-    <%= error_tag f, :company_url %>
+    <%= input form, :company_url %>
```

Comment:
It didn't need it to begin with.

That came from the default scaffold b/c I wanted quick views.

Good looking out!